### PR TITLE
Convert TableWorkspace to MaskWorkspace

### PR DESCRIPTION
**Pull request by jmborr**
**Description of work.**
- [x] `load_calibration_set` modified
- [x] unit test
- [x] release notes
- [x] PR against `ornl-next`

**To test:**  
In the analysis machines, run the script in Mantid workbench:
```python
# import mantid algorithms, numpy and matplotlib
from mantid.simpleapi import *
import matplotlib.pyplot as plt
import numpy as np
from corelli.calibration import load_calibration_set, apply_calibration

Load(Filename='CORELLI_165752-165763',  # Silicon powder runs
     OutputWorkspace='Si',
     LoadLogs=False)
Rebin(InputWorkspace='Si', OutputWorkspace='Si', Params=[3000,13600,16660], PreserveEvents=False)  # deallocate RAM

load_calibration_set(input_workspace='Si',
                     database_path='/SNS/CORELLI/shared/calibration/tube',
                     output_calibration_name='tube_calibration',
                     output_mask_name='tube_mask')

apply_calibration(workspace='Si', output_workspace='Si', calibration_table='tube_calibration')
MaskDetectors(Workspace='Si', MaskedWorkspace='tube_mask')
```
Workspace `Si` has a few tubes masked. They can be seen with the intrument viewer.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: Framework/MDAlgorithms/src/IntegrateQLabEvents.cpp [hunk 1]
```diff
@@ -301,6 +316,27 @@ PeakShapeEllipsoid_const_sptr IntegrateQLabEvents::ellipseIntegrateEvents(
...
+             << "back_outer_radius = " << back_outer_radius << "\n"
+             << "sigmas = (" << sigmas[0] << "," << sigmas[1] << "," << sigmas[2] << ")\n"
+             << "r1, r2, r3 = " << r1 << "," << r2 << "," << r3 << "\n"
+             << "peak_w_back.first = " << peak_w_back.first << "\n"
+             << "backgrd.first = " << backgrd.first << "\n"
+             << "ratio = " << ratio << "\n"
+             << "inti = peak_w_back.first - ratio * backgrd.first = " << inti << "\n";
+    g_log.debug() << debugmsg.str();
+  }
+
```
Comment (KedoKudo) [inline Framework/MDAlgorithms/src/IntegrateQLabEvents.cpp:339]: Could you rebase the branch to current ornl-next?  I don't think the changes here is relevant to this PR.

**Review by KedoKudo — DISMISSED** [review]
The testing script produces results as described, and the modification is straight forward.
The commit history is a little bit confusing as it contains something that is not related to this PR.
Functionality wise, I think this PR is good for merging, however, I would still recommend fixing the commit history via rebasing if possible.

**Review by KedoKudo — APPROVED** [review]

File: Framework/MDAlgorithms/src/IntegrateQLabEvents.cpp (hunk 1 above)
Comment (jmborr) [reply Framework/MDAlgorithms/src/IntegrateQLabEvents.cpp]: I just did. The changes from IntegrateQLabEvents.cpp are gone. You will have to give your approval again, though

Comment (KedoKudo) [reply Framework/MDAlgorithms/src/IntegrateQLabEvents.cpp]: already did :sunglasses:

**Pull request merged**